README.md

```diff
@@ -141,6 +141,37 @@ $ espercli token show -j
 {"Enterprise": "f44373cb-1800-43c6-aab3-c81f8b1f435c", "Developer App": "5b4ececb-b446-4f47-9e6f-0b47760763be", "Token": "U1XEFTNS1ujAMK2Q7Gl3hfPclCclhX", "Expires On": ["read", "write", "update", "introspection", "sdk", "register"], "Created On": "2019-08-20 08:02:16.640250+00:00", "Updated On": "2019-08-20 08:02:16.640275+00:00"}%
 ```
 
+#### 2. renew
+Renew token
+```sh
+$ espercli token renew [OPTIONS]
+```
+##### Options
+| Name, shorthand     | Default | Description|
+| --------------------|:-------:|:-----------|
+| --developerapp, -d  |         | Developer App Id |       
+| --token, -t         |         | Token to renew |
+| --json, -j          |         | Render result in JSON format |
+
+##### Example
+```sh
+$ espercli token renew -d 5fa87c46-bef7-4c1f-9ca8-ebf022d118b6 -t mzbrCDAKVyHcnNye7zdYuuLVVr22Pm
+
+TITLE          DETAILS
+Id             570
+User           bindya
+Enterprise Id  Esper [f44373cb-1800-43c6-aab3-c81f8b1f435c]
+Developer App  DeveloperApp object (5fa87c46-bef7-4c1f-9ca8-ebf022d118b6)
```

Comment:
don't use string representation of the developer app object, use developer app id

Reply:
This is how the API returns. I'll extract the id from it.